mantidproject/mantid
# RemoveMaskedSpectra algorithm.

[11619](http://trac.mantidproject.org/mantid/ticket/11619)

**To test**
- Check that CropWorkspace and ExtractSingleSpectrum still work.
- See the doc files for usage examples.
- Open the instrument view with some data.
- Go to the mask tab and select some detectors.
- Save selection as ROI to a mask workspace.
- Use this mask workspace in RemoveMaskedSpectra to extract spectra with the selected detectors.
- Check that the output workspace contains only spectra from the selection.
